config/groups.go

```diff
@@ -95,14 +95,14 @@ var GroupMap = map[string]GroupKindCalculator{
 	"aws_cloudtrail":                                           ReplaceGroupWords("cloudtrail", 0),
 	"aws_cloudwatch_event_api_destination":                     ReplaceGroupWords("events", 2),
```

Comment:
I will suggest a different group name here as the group name `events` is, in my opinion, too generic. Because we have previously put the `aws_cloudwatch_log_*` resources into the `cloudwatchlogs` group, I would suggest `cloudwatchevents` as the group name for these resources:
```suggestion
	"aws_cloudwatch_event_api_destination":                     ReplaceGroupWords("cloudwatchevents", 2),
```